src/components/Person.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import './person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        { name ? `My name is ${name}` : 'Unknown'}
+      </h2>
+      <p className="Person__age">
+        { age && `I am ${age}`}
+      </p>
+      <p className="Person__partner">
+        {(!isMarried && 'I am not married')
+        || (`My ${sex === 'm' ? 'wife' : 'husband'}'s name is ${partnerName}`)}
+      </p>
```

Comment:
This code looks too complicated
Create separate variables before return statement and store here marriage status
const personSex = sex === 'm' ? ... : ...;
const marriageStatus = `My ... name is ...`;

and use this variable in jsx

Reply:
Fixed